src/controllers/products.js

```diff
@@ -1,18 +1,31 @@
 function getAll(req, res) {
-    res.send('ALL products');
+    req.context.models.Product.findAll()
+        .then(products => res.send(products))
+        .catch(err => res.send(err));
 }
 
 function get(req, res) {
-    res.send(`SINGLE product by id: ${req.params.id}`);
+    req.context.models.Product.findAll({where: {name: req.params.name}})
+        .then(product => res.send(product))
+        .catch(err => res.send(err));
 }
 
-
 function add(req, res) {
-    res.send('ADDED product');
+    req.context.models.Product.create({
+        name: req.params.name,
+    }, {})
+        .then(product => res.json(product))
+        .catch(err => res.send(err));
 }
 
 function getReviews(req, res) {
-    res.send(`Product's with id: ${req.params.id} reviews`);
+    req.context.models.Review.findAll({
+        where: {
+            productId: req.params.id
```

Comment:
getting request parameters should be also abstracted. For example in future you'd definitely like to add validation and that would be nice place to add it

Reply:
resolve in the next task